app/scrapers/schleswig_holstein_landtag_scraper.rb

```diff
@@ -104,10 +165,51 @@ class Detail < DetailScraper
 
     def scrape
       search_url = SEARCH_URL + '(' + CGI.escape('WP=' + @legislative_term.to_s + ' AND DART=D AND DNR=' + @reference.to_s) + ')'
-      mp = mechanize.get search_url
+      m = mechanize
+      mp = m.get search_url
       table = SchleswigHolsteinLandtagScraper.extract_table(mp)
       block = SchleswigHolsteinLandtagScraper.extract_blocks(table).first
-      SchleswigHolsteinLandtagScraper.extract_paper(block)
+      extract_paper(block, mp)
     end
+
+    def extract_paper(block, mp)
+      if SchleswigHolsteinLandtagScraper.major?(block)
+        paper = SchleswigHolsteinLandtagScraper.extract_major_paper(block)
+        mp = mp.link_with(:text => /Vorgang/).click
+        paper = SchleswigHolsteinLandtagScraper.update_major_details(paper, mp)
+        return paper
+      end
+      paper = SchleswigHolsteinLandtagScraper.extract_minor_paper(block)
+      paper = SchleswigHolsteinLandtagScraper.update_minor_details(paper, mp)
+      return paper
+    end
+  end
+
+  def self.update_major_details(paper, page)
+    block = extract_detail_block page
+    line = extract_originator_line block
+    originators = NamePartyExtractor.new(line).extract
+    paper[:originators] = originators
+    paper
+  end
+
+  def self.extract_originator_line(detail_table)
+    detail_table.children.each do |child|
+      child.content.split("\n").each do |line|
+        line = line.gsub(/\p{Z}+/, ' ').strip
+        if line.start_with?('Große Anfrage')
+          return line.sub('Große Anfrage', '').strip
+        end
+      end
+    end
+  end
+
+  def self.update_minor_details(paper, page)
```

Comment:
Unused method argument - `page`. If it's necessary, use `_` or `_page` as an argument name to indicate that it won't be used.